explain/basics-of-postgres-query-planning.mdx

```diff
@@ -69,4 +69,4 @@ Here we can see that the query did in fact return a single row, and it took us 0
 
 Obviously EXPLAIN plans can be a lot more complex. We've compiled a repository of the different [scan nodes](/docs/explain/scan-nodes), [join nodes](/docs/explain/join-nodes), and [other nodes](/docs/explain/other-nodes) in the documentation to help you when reading a plan.
 
-To make collection and interpretation of plans easier, you can utilize **[auto_explain](/docs/log-insights/setup/auto_explain)** and **[pganalyze EXPLAIN Insights](/docs/explain/insights)** to get automated insights into what can be optimized about a query plan.
\ No newline at end of file
+To make collection and interpretation of plans easier, you can utilize **[auto_explain](/docs/setup/auto_explain)** and **[pganalyze EXPLAIN Insights](/docs/explain/insights)** to get automated insights into what can be optimized about a query plan.
```

Comment:
Looks like this is missing part of the path: https://pganalyze.com/docs/explain/setup/auto_explain